BugReproductionExample/Components/CodeMirror.razor

```diff
@@ -0,0 +1,47 @@
+﻿@using GaelJ.BlazorCodeMirror6
+@using GaelJ.BlazorCodeMirror6.Models
+
+
+<CodeMirror6Wrapper
+    IsWASM="false"
+    @bind-Doc="Text"
+    TabSize="@TabSize"
+    IndentationUnit="@TabSize"
+    Theme="@Theme"
+    Language="@Language"
+    AutoFormatMarkdown="@AutoFormatMarkdown"
+    LineWrapping="@LineWrapping"
+    ReadOnly="@ReadOnly"
+    FullScreen="@FullScreen"
+    Height="50em">
+</CodeMirror6Wrapper>
+
+@code {
+    private string _text;
+
+    [Parameter]
```

Comment:
This is not the ideal way to pass a binding through in a wrapper component. Consider this instead:

![image](https://github.com/user-attachments/assets/60161b58-5c21-4485-9501-1edeb1df66b3)